### ISSUE TITLE
Fringe page

### ISSUE THREAD

**Issue by edrushka** [mozilla/mozillafestival.org#656]
https://mozillafestival.org/fringe
- [ ] ~~Add new text about the hub TBD @Saallen~~
- [ ] remove bold text 'We are expanding the festival in 2017 to feature an entire week of Fringe Events. Contact us to learn more.'
- [ ] add new copy in comment below

**Comment by edrushka** [comment]
@Saallen please share link to Fringe event form

**Comment by Saallen** [comment]
@edrushka - @mmmavis  made this form last year-
There is more blurb here needed for the MozFest hub piece so this page should not go live until we confirm all pieces are in place

**Comment by edrushka** [comment]
@Saallen OK. I added the new text to the list in the first comment. Can you please add that to your to-do list?

**Comment by edrushka** [comment]
This page can be updated as soon as the copy and form are ready.

**Comment by edrushka** [comment]
@Saallen do you have the line or two of text about the pre-week and the link to the fringe page?

**Comment by mmmavis** [comment]
@edrushka @Saallen 

Just to confirm these are the fields we want to have on the form

- Name of Event (required)
- Date and Time (required)
- Event Location (required)
- Description of your Event (required)
- Link to register for the event if available
- Privacy Policy checkbox (I’m okay with Mozilla handling this info as you explain in your [privacy policy](https://www.mozilla.org/privacy/). Please note: the information in this form will be submitted via a private Google spreadsheet.) (required)

**Comment by mmmavis** [comment]
@edrushka @Saallen I cleaned up last year's form a bit. Let me know what I need to change on the page. I'm thinking maybe we should show the form first and display the list of approved fringe events after. Thoughts?

- Here's the link to see my progress:  https://mozillafestival-org-sta-pr-666.herokuapp.com/fringe
- Spreadsheet to control what shows up on the Fringe page (mark column 'approved' with `y`): https://docs.google.com/spreadsheets/d/12RicgjS3FLz9aIZkxuqfMCFnooFOrc3Np2qP4ezdPhk/edit#gid=1651772276

**Comment by mmmavis** [comment]
@edrushka can you help me revise copy on this screen?

<img width="819" alt="screen shot 2017-07-26 at 5 42 04 pm" src="https://user-images.githubusercontent.com/2896608/28649459-d6745d64-7229-11e7-87ee-e21e7870b3ad.png">

**Comment by Saallen** [comment]
Hi @mmmavis last night 
@edrushka , myself and chris had a chat. In order for an event to be added to the MozFest Fringe page- it must meet a small set of criteria.
We are writing this up now so it can be added to the top of the page.

**Comment by edrushka** [comment]
Please investigate displaying events in columns (3 columns) @mmmavis

**Comment by edrushka** [comment]
Update: I'm nudging the copy decisions along 😉  This work is moving, but most likely won't be ready for today.

**Comment by mmmavis** [comment]
@edrushka I have [this PR](https://github.com/mozilla/mozillafestival.org/pull/666) open and a temp Heroku instance running to get feedback from: https://mozillafestival-org-sta-pr-666.herokuapp.com/fringe . I won't push this PR live until we are good on the content. 

(Note that 3-column cards look too narrow so I changed them to 2-column.)

**Comment by edrushka** [comment]
Unblocked! Here's the updated copy for the Fringe page. I'll make a separate ticket for updates to the form.

----------

If MozFest wasn’t enough for you…
MozFest is about working in the open and sharing ideas with a global network of people dedicating to protecting the health of the Internet.

There are innovative events taking place around the world with this same philosophy, and we would love to exchange ideas and celebrate together. If your event demonstrates some of the following characteristics, we’d love to promote it as a MozFest Fringe event, and share some of our resources with you:

- Event is co-designed with partners.
- Includes creative, hands-on activities.
- Facilitation promotes collaboration, innovation and respect.
- Designed in the open (e.g. you publish process blogs or have an open curation process).
- Presents leadership opportunities for all (attendees, mentors, event staff).
- Promotes protects or advances the health of the Internet.
- Includes a code of conduct, ensuring all are welcome to share and be heard.

Add your event to the MozFest Fringe calendar.
[button: Add event]

**Comment by mmmavis** [comment]
Done: https://mozillafestival.org/fringe Backend spreadsheet has been shared with @Saallen and @edrushka 🍰 🍰 🍰
